auth/user_mgt.go

```diff
@@ -331,6 +340,7 @@ const (
 	unauthorizedContinueURI  = "unauthorized-continue-uri"
 	unknown                  = "unknown-error"
 	userNotFound             = "user-not-found"
+	maximumUserCountExceeded = "maximum-user-count-exceeded"
```

Comment:
This shouldn't be needed. We don't use error codes for developer errors.

Reply:
Done.